addons/im_livechat/static/tests/thread_model_patch.test.js

```diff
@@ -46,6 +47,28 @@ test("Thread name unchanged when inviting new users", async () => {
     await contains(".o-mail-Discuss-threadName[title='Visitor #20']");
 });
 
+test("Thread name renamed after editing", async () => {
```

Comment:
Would rename test to "Can set a custom name to livechat conversation".
For clarity that this uses "custom" name rather than conversation name, which is a relevant nuance functionally.